### ISSUE TITLE
Bug: while harvesting, ValidationError: Error creating geometry: context does not provide geo interface

### ISSUE THREAD

**Issue by woodt** [GSA/datagov-ckan-multi#319]
```
2020-06-04 20:52:03,047 INFO  [ckanext.harvest.harvesters.base] Package with GUID acumar_a9a46446-7a18-494b-8166-4f17e65f7ce9 does not exist, let's create it
2020-06-04 20:52:03,093 INFO  [ckanext.harvest.queue] Received harvest object id: 166df129-71d3-48f5-b853-bf2fe9b6854a
2020-06-04 20:52:03,117 DEBUG [ckanext.harvest.harvesters.ckanharvester] In CKANHarvester import_stage
2020-06-04 20:52:03,169 INFO  [ckanext.harvest.harvesters.base] Package with GUID acumar_b54dcccd-5efa-4908-b24e-a8c837b4e55d does not exist, let's create it
2020-06-04 20:52:03,291 DEBUG [ckanext.spatial.plugin] Received: u'["ARG", "02"]'
2020-06-04 20:52:03,297 ERROR [ckanext.harvest.harvesters.base] {'spatial': [u'Error creating geometry: Context does not provide geo interface']}
Traceback (most recent call last):
  File "/usr/lib/ckan-new/src/ckanext-harvest/ckanext/harvest/harvesters/base.py", line 372, in _create_or_update_package
    else 'package_create_rest')(context, package_dict)
  File "/usr/lib/ckan-new/src/ckan/ckan/logic/__init__.py", line 466, in wrapped
    result = _action(context, data_dict, **kw)
  File "/usr/lib/ckan-new/src/ckan/ckan/logic/action/create.py", line 209, in package_create
    item.create(pkg)
  File "/usr/lib/ckan-new/src/ckanext-spatial/ckanext/spatial/plugin.py", line 90, in create
    self.check_spatial_extra(package)
  File "/usr/lib/ckan-new/src/ckanext-spatial/ckanext/spatial/plugin.py", line 125, in check_spatial_extra
    raise p.toolkit.ValidationError(error_dict, error_summary=package_error_summary(error_dict))
ValidationError: {'spatial': [u'Error creating geometry: Context does not provide geo interface']}
```

## How to reproduce

Attempt to harvest CKAN harvest source at https://datos.gob.ar in the *sandbox*

## Expected behavior

Harvest succeeds

## Actual behavior

Error, see stack trace above.

**Comment by woodt** [comment]
See datagov-ckan-multi/#303

**Comment by kimwdavidson** [comment]
Let's take another look after https://github.com/GSA/catalog.data.gov/issues/55 is completed

**Comment by avdata99** [comment]
This is error was fixed by [Catalog#55](https://github.com/GSA/catalog.data.gov/issues/55)
We skipped the `spatial` validation for 55 (as we do for classic catalog CKAN 2.3).  

The `check_spatial_extra` function is skipped so this error does not exist anymore

```
def check_spatial_extra(self,package):
for extra in package.extras_list:
    if extra.key == 'spatial':
        try:
            geometry = json.loads(extra.value)
        except:
************** ERROR IN 55 *******************
            error_dict = {'spatial':[u'Error decoding JSON object: %s' % str(e)]}

        try:
            save_package_extent(package.id,geometry)
        except ValueError,e:
************** ERROR IN THIS ISSUE *******************
            error_dict = {'spatial':[u'Error creating geometry: %s' % str(e)]}
```
I restart the harvest job to validate [here](https://catalog-next.sandbox.datagov.us/harvest/admin/4b26cadc-df44-479d-8934-e3d8ea49b889)

We can QA this issue

**Comment by adborden** [comment]
Verified with the [Datos Argentina](https://catalog-next.sandbox.datagov.us/harvest/admin/4b26cadc-df44-479d-8934-e3d8ea49b889) harvest source.
